app/views/root/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Root#index</h1>
+<div id="root"></div>
```

Comment:
- [ ] the Api working fine also all React stuff are there but when I tried to  run the application I got this error please make sure to fix that in order for we can review your app properly 
- The Error 
<img width="1089" alt="Screen Shot 2022-10-08 at 11 49 43" src="https://user-images.githubusercontent.com/61790587/194703901-e9ea01a0-8b32-406e-b50d-4dc7a1c616b2.png">

Reply:
how did you run the app, so that i can have an idea of how to reproduce that error, :eyes: , i don't have it on my system

Reply:
@KanoCode I run it with `./bin/dev`